elisp/intero.el

```diff
@@ -1455,6 +1495,11 @@ Completions for PREFIX are passed to CONT in SOURCE-BUFFER."
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;; Process communication
 
+(defun intero-call-process (program &optional infile destination display &rest args)
+  "Synchronously call a process. Same interface as call-process/process-file."
+  (let ((process-args (append (list program infile destination display) args)))
+    (apply 'process-file process-args)))
```

Comment:
I think I know the reason for this function, it ensures a tramp process is created instead of a local one when necessary. But perhaps it should be documented. It was sensible to make an abstraction function for this. :+1: